shared-ui/components/meet-the-team/meet-all-core/MeetAllCore.styles.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import { P, H3 } from '../../../style/typography';
+import { colors } from '../../../style/colors';
+import { max } from '../../../lib/responsive';
+
+const StyledMeetCoreContainer = styled.div`
+  display: flex;
+  margin: auto;
+  padding: 8em;
+  @media ${max.tabletLg} {
+    padding: 4em;
+  }
+`;
+
+const StyledOurTeamCore = styled(P)`
+  color: ${colors.WHITE};
+  text-align: left;
+  padding-top: 3.5em;
+`;
+
+const StyledHeaderCore = styled(H3)`
+  text-align: left;
+`;
+
+const StyledDescriptionTeamCore = styled(P)`
+  color: ${colors.TEXT_BROWN};
+  text-align: left;
+  padding-top: 1em;
```

Comment:
Can we add

```
  @media ${max.tabletLg} {
    padding-bottom: 1em;
  }
```

here? I think it'd be nice to add a bit more padding / separation!